qa/integration/specs/logstash_to_logstash_spec.rb

```diff
@@ -56,7 +56,8 @@ def run_logstash_instance(config_name, options = {}, &block)
                                     "--pipeline.id", config_name,
                                     "--path.config", config_to_temp_file(@fixture.config(config_name, options)),
                                     "--path.data", get_temp_path_dir,
-                                    "--api.http.port", api_port.to_s)
+                                    "--api.http.port", api_port.to_s,
+                                    "-r")
```

Comment:
Maybe prefer full setting for readability. Is `-r` `--config.reload.automatic`?

Reply:
Yup, just replaced. My instinct to always use shortest form :)